src/api/generatedTypes.ts

```diff
@@ -2104,6 +2137,7 @@ export enum Topics {
   Food = 'FOOD',
   Gaming = 'GAMING',
   HealthFitness = 'HEALTH_FITNESS',
+  Home = 'HOME',
```

Comment:
i unfortunately think we need to manually remove this line. will give context in slack.

Reply:
actually - i'll let @nina-py chime in here to confirm if we have to remove this line or not.

Reply:
It's ok to keep it in in the generated types as long as we're not using it in frontend code itself - this PR is waiting and has that change: https://github.com/Pocket/curation-admin-tools/pull/1212/files